zlint/zlint.go

```diff
@@ -5,13 +5,37 @@
 package zlint
 
 import (
+	"bytes"
 	"encoding/base64"
+	"encoding/json"
 	"errors"
+	"fmt"
 	"github.com/zmap/zcrypto/x509"
 	"github.com/zmap/zlint/lints"
 	"time"
 )
 
+type PrettyOutput struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
+	Providence  string `json:"providence"`
+}
+
+//Pretty Print lint outputs
+func PrettyPrintZLint() {
+	for _, l := range lints.Lints {
+		p := PrettyOutput{}
+		p.Name = l.Name
+		p.Description = l.Description
+		p.Providence = l.Providence
+		out, _ := json.Marshal(p)
+		out = bytes.Replace(out, []byte("\\u003c"), []byte("<"), -1)
+		out = bytes.Replace(out, []byte("\\u003e"), []byte(">"), -1)
+		out = bytes.Replace(out, []byte("\\u0026"), []byte("&"), -1)
```

Comment:
This isn't really a pretty print, given that's it's outputting minified JSON objects, so I'd say just leave it.